ui/app/pages/swaps/fee-card/fee-card.js

```diff
@@ -83,26 +165,40 @@ export default function FeeCard({
           </div>
         </div>
         {!hideTokenApprovalRow && (
-          <div className="fee-card__top-bordered-row">
+          <div className="fee-card__row-header">
             <div className="fee-card__row-label">
               <div className="fee-card__row-header-text">
                 {t('swapThisWillAllowApprove', [tokenApprovalTextComponent])}
               </div>
-              <div
-                className="fee-card__link"
-                onClick={() => onTokenApprovalClick()}
-              >
-                {t('swapEditLimit')}
+              <div className="fee-card__info-tooltip-container">
```

Comment:
could we make use of the wrapperClassName/containerClassName prop on the InfoTooltip component instead of manually creating the extra div here?

Reply:
some day I will do this without you having to ask me about it on a PR :sweat_smile:

Reply:
Done in a7d49f261